supabase/functions/_backend/triggers/cron_hourly_storage.ts

```
@@ -0,0 +1,380 @@
+        const positiveData: typeof data = []
+        const negativeData: typeof data = []
+        for (const item of data) {
+            if (item.size > 0) {
+                positiveData.push(item)
+            }
+            else {
+                negativeData.push(item)
+            }
+        }
+
+        semiSortedData.push(...positiveData)
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)